src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrderStaticGroupSeperated.java

```diff
@@ -0,0 +1,10 @@
+package com.puppycrawl.tools.checkstyle.checks.imports.importorder;
+
+import static java.lang.Math.*;
+
+import static org.junit.Assert.assertEquals;
```

Comment:
please make test case for more than one empty line separator.

Reply:
Done.